cmd/subnetcmd/join.go

```diff
@@ -277,9 +285,95 @@ func joinCmd(_ *cobra.Command, args []string) error {
 
 	ux.Logger.PrintToUser("VM binary written to %s", vmPath)
 
+	if forceWrite {
```

Comment:
When would we not want to write this?

Reply:
the join command originally presented the user the choice of automatic changing a node conf, or manual chaging it (printing the changes to be made to the user which himself made the change). the original command only change the tracked subnets in an avago conf.
I just kept part of the original spirit of the command and only make an avago  node conf change if the used explicit  asks to do so by using the --force-write flag
to fully continue with that spirit, if not force write, the command should tell the user where to put all the confs, but I opted to do that in a subsequent step